hydroshare/settings.py

```diff
@@ -718,3 +722,161 @@
 
 # detect test mode to turn off some features
 TESTING = len(sys.argv) > 1 and sys.argv[1] == 'test'
+
+REDIS_HOST = os.environ.get('REDIS_PORT_6379_TCP_ADDR', 'localhost')
+REDIS_PORT = 6379
+
+REDIS_CONNECTION = redis.Redis(
+    host=REDIS_HOST,
+    port=REDIS_PORT,
+    db=4)
+WMS_CACHE_DB = redis.Redis(
+    host=REDIS_HOST,
+    port=REDIS_PORT,
+    db=5)
+PERMISSIONS_DB = redis.Redis(
+    host=REDIS_HOST,
+    port=REDIS_PORT,
+    db=6)
+
+RABBITMQ_HOST = os.environ.get('RABBITMQ_PORT_5672_TCP_ADDR', 'localhost')
+RABBITMQ_PORT = '5672'
+
+# celery settings
+# customizations: we need a special queue for broadcast signals to all
+# docker daemons.  we also need a special queue for direct messages to all
+# docker daemons.
+BROKER_URL = 'amqp://guest:guest@{RABBITMQ_HOST}:{RABBITMQ_PORT}//'.format(RABBITMQ_HOST=RABBITMQ_HOST,
+                                                                           RABBITMQ_PORT=RABBITMQ_PORT)
+CELERY_ACCEPT_CONTENT = ['json']
+CELERY_TASK_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'json'
+CELERY_DEFAULT_QUEUE = 'default'
+DEFAULT_EXCHANGE = Exchange('default', type='topic')
+
+CELERY_QUEUES = (
+    Queue('default', DEFAULT_EXCHANGE, routing_key='task.default'),
+)
+CELERY_DEFAULT_EXCHANGE = 'tasks'
+CELERY_DEFAULT_EXCHANGE_TYPE = 'topic'
+CELERY_DEFAULT_ROUTING_KEY = 'task.default'
+CELERY_ROUTES = ('hs_core.router.HSTaskRouter',)
+
+# DEVELOPMENT EXAMPLE ONLY
+# Make these unique, and don't share it with anybody
+SECRET_KEY = "9e2e3c2d-8282-41b2-a027-de304c0bc3d944963c9a-4778-43e0-947c-38889e976dcab9f328cb-1576-4314-bfa6-70c42a6e773c"
+NEVERCACHE_KEY = "7b205669-41dd-40db-9b96-c6f93b66123496a56be1-607f-4dbf-bf62-3315fb353ce6f12a7d28-06ad-4ef7-9266-b5ea66ed2519"
+
```

Comment:
@cuahsimarko Even though these keys are for development example only, I think they belong to local_settings.py since they need to be changed for production with the correct keys. I believe settings.py should hold all settings that apply to all environments, all environment-specific settings need to go to local_settings.py.